src/blender_mcp/server.py

```diff
@@ -193,7 +193,6 @@ async def server_lifespan(server: FastMCP) -> AsyncIterator[Dict[str, Any]]:
 # Create the MCP server with lifespan support
 mcp = FastMCP(
     "BlenderMCP",
-    description="Blender integration through the Model Context Protocol",
```

Comment:
or even better using `instructions` as stated in their docs https://gofastmcp.com/servers/server#param-instructions